modules/test/playwright/pages/segments-web/SegmentEditorPage.ts

```diff
@@ -36,7 +36,12 @@ export class SegmentEditorPage {
 	 * for each section so nesting is not supported
 	 */
 
-	async createSegment(name: string, properties: SegmentProperties) {
+	async createSegment(
+		name: string,
+		configure: boolean,
+		submit: boolean,
+		properties: SegmentProperties
+	) {
```

Comment:
are we the only one that are using this? because you changed the parameter order so we need to update it everywhere (and I don't get why you put the new params in the middle)

Reply:
also we can think to put them at the end with a default value so we don't need to always pass a value, we pass it only when it's not the default one

Reply:
Nice catch. will address it. We are basically the only ones using this other than the segment team, and that portion is addressed in this PR.

Reply:
I think you can pass the frame to the method. Should be already done in accountUsers.spec.ts or accountOrganizations.spec.ts